### PR TITLE
Rename master to main

### PR DESCRIPTION
**Pull request by jesserizzo**
**Pull request recommendations:**
- [ ] Name your pull request _your-development-type/short-description_. Ex: _feature/read-tiff-files_
- [ ] Link to any relevant issue in the PR description. Ex: _Resolves [gh-12], adds tiff file format support_
- [ ] Provide context of changes.
- [ ] Provide relevant tests for your feature or bug fix.
- [ ] Provide or update documentation for any feature added by your pull request.

Thanks for contributing!

### CODE REVIEW

File: .github/workflows/build-docs.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ name: Documentation
 on:
   push:
     branches:
-      - main
+      - main # Should this be running?
```
Comment (jesserizzo) [inline .github/workflows/build-docs.yml:6]: This was already triggered from main, so does that mean it's not ready to be run? It is failing.

File: .github/workflows/build-main.yml [hunk 2]
```diff
@@ -3,7 +3,7 @@ name: Build Main
 on:
   push:
     branches:
-      - main
+      - main # Should this be running?
```
Comment (jesserizzo) [inline .github/workflows/build-main.yml:6]: This one doesn't fail, but still I'm not sure if it should be run since it wasn't before.

**Comment by jesserizzo** [comment]
@gtdiehl Any idea why some of these workflows were already triggered from 'main'?

**Comment by gtdiehl** [comment]
@jesserizzo I don't know maybe some timing issue? It appears everything is passing after changing the test

**Comment by jesserizzo** [comment]
@bdraco It looks like you just added build-docs and build-main a few days ago. Both triggered off of `main` even though the default branch at that time was named `master`. Now after the name change, build-main succeeds, but build-docs fails. Any thoughts?

**Comment by gtdiehl** [comment]
@jesserizzo I've disabled the Documentation workflow for now. I thought this was a builtin app/tool from Github, but I think we need an extra ```Makefile``` to utilize the ```gen-docs``` target

File: .github/workflows/build-docs.yml (hunk 1 above)
Comment (gtdiehl) [reply .github/workflows/build-docs.yml:6]: Make the change to ```main``` but I have manually disabled the ```Documentation``` workflow for now

File: .github/workflows/build-main.yml (hunk 2 above)
Comment (gtdiehl) [reply .github/workflows/build-main.yml:6]: It should be run when we do a push to ```main```

**Comment by bdraco** [comment]
> @bdraco It looks like you just added build-docs and build-main a few days ago. Both triggered off of `main` even though the default branch at that time was named `master`. Now after the name change, build-main succeeds, but build-docs fails. Any thoughts?

It's a workflow template that assumes there are docs.  Probably best to just disable it until their are

**Pull request merged**